portal/babel.cfg

```diff
@@ -0,0 +1,4 @@
+[python: **.py]
```

Comment:
As we're now setting (instance_relative_config=True)[https://github.com/uwcirg/true_nth_usa_portal/blob/develop/portal/app.py#L66] - shouldn't this config also be in the instance directory?